src/NServiceBus.Core.Tests/Recoverability/When_message_is_deferred_by_slr.cs

```diff
@@ -0,0 +1,145 @@
+﻿namespace NServiceBus.Core.Tests.Recoverability
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Threading.Tasks;
+    using NServiceBus.Pipeline;
+    using NServiceBus.Transports;
+    using NUnit.Framework;
+    using Testing;
+
+    [TestFixture]
+    public class When_message_is_deferred_by_slr
```

Comment:
I agree we could be doing more/different asserting here. The main driving force behind current once was to have tests which are testing similar scenario to ATT.